ObjectiveSee/OverSight.munki.recipe

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>Description</key>
+    <string>Downloads, unzips, verifies, packages and imports the installer of the latest version of OverSight by Objective-See into munki.</string> 
+    <key>Identifier</key>
+    <string>com.github.its-unibas.munki.OverSight</string>
+    <key>ParentRecipe</key>
+    <string>com.github.its-unibas.pkg.OverSight</string>
+    <key>MinimumVersion</key>
+    <string>1.0.0</string>
+    <key>Input</key>
+    <dict>
+        <key>MUNKI_NAME</key>
+        <string>%NAME%</string>
+        <key>MUNKI_REPO_SUBDIR</key>
+        <string>apps/%NAME%</string>
+        <key>MUNKI_CATEGORY</key>
+        <string>Utilities</string>
+        <key>MUNKI_DEVELOPER</key>
+        <string>Objective-See</string>
+        <key>DISPLAY_NAME</key>
+        <string>%NAME%</string>
+        <key>pkginfo</key>
+        <dict>
+            <key>catalogs</key>
+            <array>
+                <string>testing</string>
+            </array>
+            <key>description</key>
+            <string>OverSight monitors a mac's mic and webcam, alerting the user when the internal mic is activated, or whenever a process accesses the webcam.</string>
+            <key>display_name</key>
+            <string>%DISPLAY_NAME%</string>
+            <key>name</key>
+            <string>%MUNKI_NAME%</string>
+            <key>unattended_install</key>
+            <true/>
+            <key>category</key>
+            <string>%MUNKI_CATEGORY%</string>
+            <key>developer</key>
+            <string>%MUNKI_DEVELOPER%</string>
+            <key>postinstall_script</key>
+            <string>#!/bin/sh
+sudo "/Applications/%NAME% Installer.app/Contents/MacOS/Oversight Installer" -install
```

Comment:
Same comments as above apply:
* No need for `sudo`
* Don't use `NAME`
* Install `installer.app` in another location if possible

Reply:
I have now implemented the requested changes